# угловая скорость по курсу [град / сек]
float32 rate_head

# угловая скорость по дифференту [град / сек]
float32 rate_pitch

# угловая скорость по крену [град / сек]
float32 rate_roll 